plugins/source/plugin.go

```diff
@@ -12,6 +12,16 @@ import (
 	"golang.org/x/sync/semaphore"
 )
 
+const (
+	SchedulerDFS        = "dfs"
```

Comment:
[nit]: use a typed consts instead of strings?

Reply:
Updated 👍